cocos/editor-support/spine/SkeletonAnimation.cpp

```diff
@@ -88,6 +88,10 @@ SkeletonAnimation* SkeletonAnimation::createWithJsonFile (const std::string& ske
 	spAtlas* atlas = spAtlas_createFromFile(atlasFile.c_str(), 0);
 	node->initWithJsonFile(skeletonJsonFile, atlas, scale);
 	node->autorelease();
+    	if(!node->getSkeleton() || !node->getSkeleton()->data)
+    	{
+        	return nullptr;
```

Comment:
Here is a memory leak, you've lost a weak reference to the allocated memory before

Reply:
Thanks,

So release node and spAtlas?